src/main/java/org/app/TransactionsController.java

```diff
@@ -21,15 +22,18 @@
 
 @RestController
 class TransactionsController {
-    @GetMapping("transactions")
+    @Autowired
```

Comment:
Protip: Spring already supports dependency injection without using the `@Autowired` annotation.